File: Regular Expressions/Code/chapter6/6-16/recipe6-16.py
#!/usr/bin/python
import re
import sys

nargs = len(sys.argv)

if nargs > 1:

	myfile = sys.argv[1]
	output = open( myfile )

	regex = re.compile( r'\r' )

	lines = output.readlines()
	for line in lines:
		formatted = regex.sub( r'', line )
		print formatted,

	output.close()

else:
	print 'I\'ve seen Tron eight times',

File: Regular Expressions/Code/chapter6/6-16/recipe6-16.pl
#!/usr/bin/perl -w
use strict;

open ( FILE, $ARGV[0] ) || die "Cannot open file!";

while (<FILE>)
{ 
	my $line = $_;
	if ( $line =~ s/\r//g )
	{
		print $line;
	}
}

close( FILE );
